include/boost/url/decode.hpp

```diff
@@ -16,25 +16,82 @@
 #include <boost/url/decode_view.hpp>
 #include <boost/url/error_types.hpp>
 #include <boost/url/string_view.hpp>
+#include <boost/url/pct_string_view.hpp>
 #include <boost/url/detail/decode.hpp>
 #include <boost/url/grammar/all_chars.hpp>
 #include <boost/url/grammar/charset.hpp>
 
 namespace boost {
 namespace urls {
 
-/** Validate and decode a string view
+/** Decode a string view into a buffer.
```

Comment:
We shouldn't say "decode a string view" (the "string view" part is what I am objecting to here). We are decoding a string. And it doesn't matter that it is "into a buffer", that's implied by the parameter list. The brief can simply say "Apply percent-decoding to a string"